index.js

```diff
@@ -114,11 +137,7 @@ async function fastifyView (fastify, opts) {
       done = args.pop()
     }
 
-    let promise = !page ? Promise.reject(new Error('Missing page')) : renderer.apply(this, args)
-
-    if (minify) {
-      promise = promise.then((result) => minify(result, globalOptions.htmlMinifierOptions))
-    }
+    const promise = asyncRender.apply(fakeRequest, args)
```

Comment:
There is no need for a fakeRequest, you can use a real one.

Reply:
There is no request in scope, it's the crux of https://github.com/fastify/point-of-view/issues/394 -- this is `fastify.view` not `reply.view`.

If it seems hacky, I can drop it and add the conditional into the calls to `this.getHeader()` on line 121?

My intent here was to avoid adding a branch into the critical path of the plugin to avoid performance regression.

Reply:
Or I could just construct `this` on every request like used to happen here: https://github.com/fastify/point-of-view/commit/235f5882cc9d4768ac45c35cbbb745613f162357#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L857

Same intent: avoid creating object unnecessarily in critical path for performance reasons, though in this case it would be less impactful as this is ONLY in the critical path of `fastify.view` as opposed to _everything_. lmk what you all prefer.